mod/schema/deps/UnitCard.json

```diff
@@ -7,129 +7,145 @@
 
   "definitions": {
     "Bases": {
-      "enum": ["small", "medium", "large", "huge", "laat", "epic", "long", "snail"]
+      "enum": [ "small", "medium", "large", "huge", "laat", "epic", "long", "snail" ]
     }
   },
 
-  "additionalProperties": false,
+  "oneOf": [
```

Comment:
ugly changes, but moving to accept one of two patterns:
Normal unit formatting (unchanged)
unit injection via "content" with a json file path